liblangutil/ParserBase.h

```diff
@@ -36,7 +36,14 @@ class Scanner;
 class ParserBase
 {
 public:
-	explicit ParserBase(ErrorReporter& errorReporter): m_errorReporter(errorReporter) {}
+	/// Set @_parserErrorRecover to true for additional error
+	/// recovery.  This is experimental and intended for use
+	/// front-end tools that need partial AST information even
```

Comment:
`by front-end tools`? Just sounds like smth is missing in the sentence

Reply:
I agree and will have to think about. 

The basic idea is that something like an implementation of a  Language Server Protocol (LSP) for Solidity will want (incomplete) AST information even though the overall compilation unit is faulty.